config/locales/es.yml

```diff
@@ -94,8 +94,8 @@ es:
       invalid_order: Ocurrió un error al actualizar la prioridad del rol. Verifique los valores asignados e inténtelo nuevamente
       invalid_update: Ocurrió un error al actualizar los permisos del rol. Verifique los valores asignados e inténtelo nuevamente
       name: Nombre del rol
-      new_role: Crear un rol nuevo
-      role_has_users: "Este rol se encuentra asignado a %{user_count}cuentas de usuario. Debe borrar todas las cuentas asociadas a este rol antes de eliminarlo."
+      new_role: Crear un nuevo rol
```

Comment:
Same as before. Ok, my mistake, It can be used before or after, but in this (and the former case), the adjective is used to make a distintion of another suibject, therefore it should go after the noun.